agenta-backend/agenta_backend/migrations/postgres/data_migrations/projects.py

```diff
@@ -0,0 +1,143 @@
+import os
+import traceback
+
+import click
+from slugify import slugify
+
+from sqlalchemy.future import select
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.exc import MultipleResultsFound
+
+from agenta_backend.models.db_models import AppDB, ProjectDB
+
+
+def assign_default_project():
+    # Define the project name that was used as default
+    PROJECT_NAME = "Default"
```

Comment:
`default=True` rather that `project_name="Default"`

Reply:
Let's remove `slug_name` and let's set `name` (not `project_name`) to `default` for now, upon creation.

Reply:
Got it.